omero/features/share/index.html

```diff
@@ -0,0 +1,35 @@
+---
+layout: omero-features
+title: OMERO Features
+feature: Share
+usergroup: omero
+meta_description: Learn how to share data within OMERO.
+---
+
+        <!-- begin OMERO features -->
+        <hr class="whitespace">
+        <div class="row column text-center">
+            <h2 class="header-blue">Sharing your data in OMERO</h2>
+            <hr class="medium-10">
+            <p class="subheader">
+                Share your images and collaborate with colleagues using the permissions system.
+            </p>
+            <hr class="medium-10">
+            <div class="row small-12 medium-up-3">
+
+                <div class="omero-feature-container column text-center">
+                    <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_manage.svg">
+                    <h4>Permissions</h4>
+                    <p>Allows users to share data within groups and collaborate on their data.</p>
+                    <a class="button hollow tiny" href="https://omero-guides.readthedocs.io/en/latest/introduction/docs/data-management.html" target="_blank">View user guide</a>
+                    <a class="button hollow tiny" href="https://docs.openmicroscopy.org/omero/{{ site.omero.version }}/sysadmins/server-permissions.html" target="_blank">View sysadmin guide</a>
+                </div>
+                <div class="omero-feature-container column text-center">
+                    <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_annotate.svg">
+                    <h4>Duplicate and Move the data to Share</h4>
```

Comment:
I wonder a bit about this wording. (Something we've struggled with before). "Create cost-less duplicates of your data to share with others"

Reply:
Or just "Zero-cost duplicates"? with the rest in the text.

Reply:
I would guess that what you are asking for is to visualize Duplicate feature more. Both the titles and the short texts are quite short by the constraints of the layout. What about if we come up with a new icon in the same pane ? The Sharing aspect will be not fully understood without the Move word there, and the cross-group hint imho. But no harm in having a "kinda slightly bannery" new icon. I will play with it a bit